types/src/account_config/constants/coins.rs

```diff
@@ -3,13 +3,16 @@
 
 use crate::account_config::constants::{from_currency_code_string, CORE_CODE_ADDRESS};
 use move_core_types::{
-    identifier::Identifier,
+    ident_str,
+    identifier::IdentStr,
     language_storage::{ModuleId, StructTag, TypeTag},
 };
 use once_cell::sync::Lazy;
 
 pub const XDX_NAME: &str = "XDX";
+pub const XDX_IDENTIFIER: &IdentStr = ident_str!("XDX");
 pub const XUS_NAME: &str = "XUS";
+pub const XUS_IDENTIFIER: &IdentStr = ident_str!("XUS");
```

Comment:
Does the `ident_str` macro support things other than static strs? as in could you do `ident_str!(XUS_NAME)` or not?

Reply:
I should be able to change the macro to take `$ident:expr` instead of `$ident:literal`, which would let it accept variables.